exonum/src/runtime/mod.rs

```diff
@@ -194,11 +203,21 @@ pub trait Runtime: Send + Debug + 'static {
     /// * If panic occurs, the runtime must ensure that it is in a consistent state.
     fn stop_service(&mut self, descriptor: InstanceDescriptor) -> Result<(), ExecutionError>;
 
-    /// Execute service transaction.
+    /// Dispatch payload to the method of a specific service instance.
+    /// Service instance name and method ID are provided in the `call_info` argument and
+    /// interface name is provided as the corresponding field of the `context` argument.
```

Comment:
Why isn't the interface name (required for dispatch) the part of the call info?

Reply:
Because at the moment, "services interfaces" is private a feature and thus, service writers are limited in their ability to create their own interfaces.

Cast @popzxc to more explanations.

Reply:
Okay, are there any plans on moving it into `CallInfo`? We need to know to plan changes in EJB

Reply:
I see, but the Runtime trait isn't for service developers, it is for Runtime developers.

Reply:
> Okay, are there any plans on moving it into `CallInfo`? We need to know to plan changes in EJB

Yes, we have such plans.